realsense/common/win/common_utils.cc

```diff
@@ -49,6 +49,18 @@ scoped_ptr<base::ListValue> CreateErrorResult(ErrorCode error,
   return create_results.Pass();
 }
 
+scoped_ptr<base::ListValue> CreateDOMException(const std::string& message,
+                                               ErrorName name) {
```

Comment:
The order follows the constructor of `DOMException` (https://heycam.github.io/webidl/#es-DOMException-call):
```
DOMException(message, name)
```